app/app.go

```diff
@@ -461,6 +456,19 @@ func New(
 	)
 	parentModule := ibcparent.NewAppModule(&app.ParentKeeper)
 
+	// child keeper satfies the staking keeper interface
+	// of the slashing module
+	app.SlashingKeeper = slashingkeeper.NewKeeper(
+		appCodec,
+		keys[slashingtypes.StoreKey],
+		app.ChildKeeper,
```

Comment:
@AdityaSripal if you'd have an idea how to not have the parent keeper also uses this custom slashing keeper please lmk.